napari_aicsimageio/core.py

```diff
@@ -41,6 +63,77 @@ def _get_full_image_data(img: AICSImage, in_memory: bool) -> Optional[xr.DataArr
     return None
 
 
+# Function to handle multi-scene files.
+def _get_scenes(img: AICSImage, in_memory: bool) -> Optional[xr.DataArray]:
```

Comment:
The typing on the return is incorrect here. This function always returns `None`

Change to:
```python
-> None:
```

Reply:
Of course. Copy-pasta from your original function. Thank you!

Reply:
this approach of opening up a widget in the reader hookspec is the only good workaround I currently know of to solicit more feedback from the user, so I think it's fine to go ahead with it for now (In fact, I do the same thing in `napari-omero` to let the user log-in to their server) ... However, you should probably put a comment here that this is _not_ really a supported approach, and may well break in the future.  

I say "go for it" ... but don't tell anyone :)   and definitely be prepared for some breakage in the future